frontend/helpers/product-list-helpers.ts

```diff
@@ -75,6 +76,12 @@ export function getFacetWidgetType(attribute: string) {
    return facetWidgetTypeMap[attribute] ?? FacetWidgetType.Menu;
 }
 
+const RefinementListValueSchema = z.string().array().nonempty();
+
+export function isValidRefinementListValue(value: unknown) {
```

Comment:
This is actually checking if it is a string array, not that it is a valid refinement list value necessarily as its not allowing strings. So maybe we could rename this?

Reply:
Since `RefinementList`s are multi-selects in algolia, they will always be an array of strings even if there is just one value.

That being said, I could be convinced that `nonempty` isn't necessary, I wasn't really sure about that piece.

Reply:
Also could specify that it returns a `boolean`